waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -9,23 +9,20 @@ import
   stew/arrayops,
   ../../utils/protobuf
 
-## Bn256 and RLN are Nim wrappers for the data types used in
-## the rln library https://github.com/kilic/rln/blob/3bbec368a4adc68cd5f9bfae80b17e1bbb4ef373/src/ffi.rs
-type Bn256* = pointer
```

Comment:
How come we don't have Bn256 anymore?

Reply:
In zerokit RLN I used as default Bn254, the same curve used by circom for generating zkSNARK proofs. Hopefully in the next future, RLN object implementation and API can be extended so that can be instantiatied with any curve from ark-ec.